### PR TITLE
[TeamQuest] Team quest cpanel home

### PR DESCRIPTION
**Pull request by raztechs**
Created the template fore the cpanel home and the tests for it.

### CODE REVIEW

File: wouso/games/teamquest/cpanel_urls.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-from django.conf.urls.defaults import *
-from django.conf import settings
+from django.conf.urls import patterns, url
 
-urlpatterns = patterns('')
\ No newline at end of file
+urlpatterns = patterns('wouso.games.teamquest.cpanel_views',
+    url(r'^$', 'quests', name='teamquest_home')
+)
```
Comment (iulianR) [inline wouso/games/teamquest/cpanel_urls.py]: Newline at end of file. PEP8

File: wouso/games/teamquest/cpanel_views.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from wouso.games.teamquest.models import TeamQuest
+
+
+class HomeView(ListView):
+    model = TeamQuest
+    context_object_name = 'quests'
+    template_name = 'teamquest/cpanel/list_quests.html'
+
+
+quests = permission_required('config.change_setting')(HomeView.as_view())
```
Comment (iulianR) [inline wouso/games/teamquest/cpanel_views.py]: Newline at end of file. PEP8

File: wouso/games/teamquest/templates/teamquest/cpanel/list_quests.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    {% empty %}
+    <tr>
+        <td colspan="5">No team quests added yet!</td>
+    </tr>
+    {% endfor %}
+</table>
+
+{% endblock %}
+
+{% endblock %}
```
Comment (iulianR) [inline wouso/games/teamquest/templates/teamquest/cpanel/list_quests.html]: Newline at end of file. PEP8

File: wouso/games/teamquest/tests.py [hunk 4]
```diff
@@ -203,3 +205,39 @@ def test_quest_status_create(self):
...
+
+
+class TeamQuestViewsTest(WousoTest):
+
+    def setUp(self):
+        self.admin = self._get_superuser()
+        self.c = Client()
+        self.c.login(username='admin', password='admin')
+
+    def test_cpanel_home_view_basic(self):
```
Comment (iulianR) [inline wouso/games/teamquest/tests.py]: It's better if you only test if the Name, Manage and the button exists (just the button string, no glyphicons/trans etc). No need to test for all columns

File: wouso/games/teamquest/cpanel_views.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+from django.contrib.auth.decorators import permission_required
+from django.views.generic import ListView
+
+from wouso.games.teamquest.models import TeamQuest
+
+
+class HomeView(ListView):
```
Comment (iulianR) [inline wouso/games/teamquest/cpanel_views.py]: It's better if you name it "QuestsView"

**Comment by iulianR** [comment]
Done reviewing! Good job so far!

File: wouso/games/teamquest/tests.py [hunk 6]
```diff
@@ -203,3 +205,39 @@ def test_quest_status_create(self):
...
+        response = self.c.get(reverse('teamquest_home'))
+
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, '<span class="glyphicon glyphicon-plus"></span>{% trans 'Add team quest' %}')
```
Comment (iulianR) [inline wouso/games/teamquest/tests.py]: It's better if you only test if the Name, Manage and the button exists (just the strings, no need to check for <th>, glyphicons, trans tags etc)

**Comment by razvand** [comment]
Make sure all tests pass.

**Pull request merged**